src/main/resources/hudson/plugins/performance/tags/captionLine.jelly

```diff
@@ -27,7 +27,7 @@
     <th>${%Http Code}</th>
     <th>${%Errors} (%)</th>
     <th>${%Average} (KB)</th>
-    <th>${%Total} (KB)</th>
+    <th>${%Maximum} (KB)</th>

```

Comment:
This is the header for the table on the results page. I forgot to rename the column on my first pass